src/core/server/plugins/discovery/plugins_discovery.test.ts

```diff
@@ -183,11 +185,47 @@ test('properly iterates through plugin search locations', async () => {
     )})`,
   ]);
 
-  expect(loggingServiceMock.collect(logger).warn).toMatchInlineSnapshot(`
-Array [
-  Array [
-    "Explicit plugin paths [${TEST_EXTRA_PLUGIN_PATH}] are only supported in development. Relative imports will not work in production.",
-  ],
-]
-`);
```

Comment:
I let you imagine what this was doing when --update-snapshot....

Reply:
Past me takes full responsibility 😧